production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json

```diff
@@ -0,0 +1,322 @@
+    "GlobalPeakWindowSum": {
+        "window_shift": 4,
+        "window_width": 8,
+        "apply_integration_correction": true
+    },
+    "timestamps_pointing": "ucts",
+    "train_gamma_src_r_deg": [
+        0,
+        Infinity
+    ],
+    "source_dependent": false,
+    "mc_nominal_source_x_deg": 0.4,
+    "mc_nominal_source_y_deg": 0.0,
+    "volume_reducer": {
+        "algorithm": null,
+        "parameters": {}
+    },
+    "calibration_product": "LSTCalibrationCalculator",
+    "LSTCalibrationCalculator": {
+        "systematic_correction_path": null,
+        "npe_median_cut_outliers": [
+            -5,
+            5
+        ],
+        "squared_excess_noise_factor": 1.222,
+        "flatfield_product": "FlasherFlatFieldCalculator",
+        "pedestal_product": "PedestalIntegrator",
+        "PedestalIntegrator": {
+            "sample_size": 10000,
+            "sample_duration": 100000,
+            "tel_id": 1,
+            "time_sampling_correction_path": null,
+            "charge_median_cut_outliers": [
+                -10,
+                10
+            ],
+            "charge_std_cut_outliers": [
+                -10,
+                10
+            ],
+            "charge_product": "FixedWindowSum",
+            "FixedWindowSum": {
+                "window_shift": 6,
+                "window_width": 12,
+                "peak_index": 18,
+                "apply_integration_correction": false
+            }
+        },
+        "FlasherFlatFieldCalculator": {
+            "sample_size": 10000,
+            "sample_duration": 100000,
+            "tel_id": 1,
+            "time_sampling_correction_path": null,
+            "charge_product": "LocalPeakWindowSum",
+            "charge_median_cut_outliers": [
+                -0.9,
+                2
+            ],
+            "charge_std_cut_outliers": [
+                -10,
+                10
+            ],
+            "time_cut_outliers": [
+                2,
+                38
+            ],
+            "LocalPeakWindowSum": {
+                "window_shift": 5,
+                "window_width": 12,
+                "apply_integration_correction": false
+            }
+        }
+    },
+    "waveform_nsb_tuning": {
+        "nsb_tuning": false,
+        "nsb_tuning_ratio": 0.52,
+        "spe_location": "lstchain/data/SinglePhE_ResponseInPhE_expo2Gaus.dat"
+    },
+    "write_interleaved_events": {
+        "DataWriter": {
+            "overwrite": true,
+            "write_images": false,
+            "write_parameters": false,
+            "write_waveforms": true,
+            "transform_waveform": true,
+            "waveform_dtype": "uint16",
+            "waveform_offset": 400,
+            "waveform_scale": 80
+        }
+    },
+    "image_modifier": {
+      "increase_nsb": true,
+      "extra_noise_in_dim_pixels": 1.327,
+      "extra_bias_in_dim_pixels": 0.41,
+      "transition_charge": 8,
+      "extra_noise_in_bright_pixels": 0.0
+    }
```

Comment:
this production seems quite similar to `20240430_v0.10.4_src9_dec3476_tuned` (same dec line and somewhat close image modifier parameter). @ToraTherese have you tried that production already (even if later more fine-tuning of the NSB is needed)?

Reply:
It is indeed very similar, just different image modifier parameter. What do you mean by using a later fine-tuning of the NSB?

Reply:
Hi @ToraTherese 
I think @morcuended is actually right here.
The difference between production `20240430_v0.10.4_src9_dec3476_tuned` and the one you are requesting is 

```
"image_modifier": {
    "increase_nsb": true,
    "extra_noise_in_dim_pixels": 1.577,
    "extra_bias_in_dim_pixels": 0.463,
    "transition_charge": 8,
    "extra_noise_in_bright_pixels": 1.892
  }
```
vs
```
"image_modifier": {
      "increase_nsb": true,
      "extra_noise_in_dim_pixels": 1.327,
      "extra_bias_in_dim_pixels": 0.41,
      "transition_charge": 8,
      "extra_noise_in_bright_pixels": 0.0
    }
```

Using directly the trained model from `20240430_v0.10.4_src9_dec3476_tuned` for src10 would probably lead to very similar results - have you tried already?

Reply:
As a reference, you may have a look at https://indico.cta-observatory.org/event/5532/contributions/45108/attachments/25665/37594/DL2_auto.pdf

The difference of nsb here (0.25 or 16%) is quite small and differences will most probably be hidden by systematics.

Reply:
Hi @ToraTherese 
Have you been able to work with existing production `20240430_v0.10.4_src9_dec3476_tuned` ?
Should #473 be closed ?